notebooks/deepdive/custom/ucaip_customjob.ipynb

```diff
@@ -0,0 +1,1857 @@
+{
```

Comment:
<p>Can you delete this "End of tutorial" cell? It's not necessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='102'/>